ui.apps/src/main/javascript/minicart/src/index.js

```diff
@@ -25,8 +25,11 @@ import { CheckoutProvider, initialState as initialCheckoutState, reducer } from
 import AuthBar from './components/AuthBar';
 
 const App = () => {
+    const storeView = document.querySelector('body').dataset.storeView || 'default';
```

Comment:
Ideally, this would be passed directly into the props via that web component magic that we used in the past.
We'll leave it like that for now, but we have to address that pretty soon as we introduce component configurations.